core/src/main/java/com/google/instrumentation/trace/SpanBuilder.java

```diff
@@ -31,11 +31,10 @@
  *   private static final Tracer tracer = Tracer.getTracer();
  *   void doWork(Span parent) {
  *     // Create a Span as a child of the given parent.
- *     try (Span span = tracer.spanBuilder(parent, "MyChildSpan").startSpan()) {
- *       span.addAnnotation("my annotation");
- *       doSomeWork(span); // Manually propagate the new span down the stack.
- *     }
- *     // "span" will be ended here.
+ *     Span span = tracer.spanBuilder(parent, "MyChildSpan").startSpan();
+ *     span.addAnnotation("my annotation");
+ *     doSomeWork(span); // Manually propagate the new span down the stack.
+ *     span.end();  // Manually end the span.
```

Comment:
Good point @adriancole this is another reason why try-with-resource may work better because users do not have to think about exceptions.